modules/ols-1-0-3-release-notes.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-1-0-3-release-notes_{context}"]
+= {ols-long} version 1.0.3
+
+{ols-official} 1.0.3 is now available on {ocp-product-title} 4.15 and later.
+
+[id="ols-1-0-3-enhancements_{context}"]
+== Enhancements
+
+The following enhancements are made with {ols-official} 1.0.3:
+
+* This release makes {ols-official} 1.0.3 generally available, and is supported on {ocp-product-title} 4.15 and later.
+
+* With this update, when you specify BYO Knowledge information sources in `OLSConfig.spec.ols.rag` specification file, the custom resource (CR) uses the following default settings:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+spec:
+  ols: 
+    rag: 
+      indexPath: /rag/vector_db
+      indexID: vector_db_index
+----
++
+Previously, you had to manually define the parameter values.
```

Comment:
This emphasizes the wrong side of the equation. The key to this change is that it is only necessary to specify the BYOK image and not mention these two parameters. Users don't need know they exist and what values they hold. All they need to know is the name of the BYOK image.

Reply:
Thanks for the comment @syedriko. I pushed a change.

I know you mentioned the customer doesn't really need to know about the two parameters, but I feel like we have to mention the defaults for them or not mention them at all. 

Instead of what was pushed to the PR do think something like the following would be better?

With this update, when you specify BYO Knowledge information sources in `OLSConfig.spec.ols.rag` specification file specify the URL path for the image file. 
+
.Example configuration
[source,yaml,subs="attributes,verbatim"]
----
spec:
  ols: 
    rag: 
      - image: quay.io/<username>/my-byok-image:latest
----

Reply:
discussed with Sergey and made the update